### PR TITLE
Markdown values for order total

### PR DESCRIPTION
**Pull request by paxmanchris**
Per issue #1. I added markdown values for order total, order subtotal and currency, to be used in the  'Checkout Scripts' setting.

### CODE REVIEW

File: includes/integration.php [hunk 1]
```diff
@@ -227,8 +227,30 @@ function print_conversion_code( $code ) {
...
     }
 
+    /**
+    * Filter the code for dynamic data like price
+    * @param string $code
+    * @return string
+    */
+    function process_markdown( $code ){
+        global $wp;
+         if( is_order_received_page() and isset($wp->query_vars['order-received']) ){
```
Comment (tareq1988) [inline includes/integration.php]: Do you need to check query vars when you are checking by `is_order_received_page()`?

File: includes/integration.php [hunk 2]
```diff
@@ -59,8 +59,8 @@ function init_form_fields() {
             ),
             'checkout' => array(
                 'title'       => __( 'Checkout Scripts', 'wc-conversion-tracking' ),
-                'description' => __( 'Adds script on the purchase success page', 'wc-conversion-tracking' ),
-                'desc_tip'    => true,
+                'desc_tip'    => __( 'Adds script on the purchase success page', 'wc-conversion-tracking' ),
```
Comment (tareq1988) [inline includes/integration.php]: `desc_tip` should be set to `true`

**Comment by tareq1988** [comment]
Can you also do the same for the single product part? So if you are adding a tracking for a single product, the price will also work.

File: includes/integration.php (hunk 2 above)
Comment (paxmanchris) [inline includes/integration.php]: Not necessarily true. If  `desc_tip` has a string value it becomes the value of the `?` as seen on 1 and the `description` gets printed under the form element, as seen on `2`, in the image below:
image

File: includes/integration.php (hunk 1 above)
Comment (paxmanchris) [inline includes/integration.php]: You are right. I will take it out.

**Comment by paxmanchris** [comment]
>Can you also do the same for the single product part? So if you are adding a tracking for a single product, the price will also work.

No problem, I pushed a new commit.

**Comment by tareq1988** [comment]
I think the shortcode for single product will not work.

Codes for single product will also be printed in checkout page, [see here](https://github.com/tareq1988/woocommerce-conversion-tracking/blob/master/includes/integration.php#L195). So they need to be taken care of in that loop.

**Comment by paxmanchris** [comment]
>I think the shortcode for single product will not work.

>Codes for single product will also be printed in checkout page, see here. So they need to be taken care of in that loop.*

Given your recent update, that does not require `single product`, is this mergeable now?

also, I merged in you updates.

**Pull request merged**